lib/recurly/client.php

```diff
@@ -62,6 +62,7 @@ class Recurly_Client
   const PATH_COUPON_REDEMPTION = '/redemption';
   const PATH_COUPON_REDEMPTIONS = '/redemptions';
   const PATH_COUPONS = '/coupons';
+  const PATH_GIFT_CARDS = '/gift_cards';
```

Comment:
I'd love to stop using these constants. It seems like it just adds complexity with little benefit over static strings. What do you think?